tests/test_legofy.py

```diff
@@ -101,6 +101,12 @@ def test_get_new_filename(self):
         new_path = legofy.get_new_filename(FLOWER_PATH, '.gif')
         self.assertTrue(new_path.endswith('_lego.gif'))
 
+    def test_lego_palettes(self):
+        """Validate lego palettes structured properly."""
+        legos = legofy.palettes.legos()
+        for palette in legos:
+            self.assertFalse(len(legos[palette]) % 3)
```

Comment:
Could also check that there are no duplicates and the number of colors per palette_mode